cmd/serverless/main.go

```diff
@@ -154,10 +154,15 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		log.Warn("An API Key has been set in multiple places:", strings.Join(apikeySetIn, ", "))
 	}
 
+	// first, generically parse all _SECRET_ARN and _KMS_KEY variable
+	for envKey, envVal := range getSecretEnvVars(os.Environ(), readAPIKeyFromSecretsManager, readAPIKeyFromKMS) {
```

Comment:
maybe we could export this to a separate function a explicit name so we know directly what this code is doing (+ a comment?)

Reply:
Done 👍